### DISCUSSION TITLE
memoria se llena en un dia en pi deploy y pi hole

### DISCUSSION THREAD

**Discussion by anmr2380** [DesktopECHO/Pi-hole-for-Android#100] [General]
saludos memoria interna del celular usando pideploy pi hole  se me lleno en un solo dia alguien sabe alguna solucion o como saber que archivo o proceso es el culpable usando uf -h por terminal de indica esto esto /dev/block/platform/13500000.dwmmc0/by-name/userdata   23gigas   2   3G    100%

**Comment by DesktopECHO** [comment]
Probably excessive log files.  
Run this to verify that is the case:
`du -d1 -h  /var/log`

**Reply by anmr2380** [reply]
syslog     messages    y kern.log      crecen rapidamente en el directorio /var/log  me huele a un error generandose en el kernel debian que opinan .....pero pihole funciona normalmente hasta lo que se puede observar  use   ls -lh para ver que archivo pesaba mas

**Reply by DesktopECHO** [reply]
Android and Debian are both using the same kernel.  Android can generate a lot of logging.
This command tells logrotate to rotate the logs hourly instead of weekly:

`sudo bash -c "grep -rl weekly /etc/logrotate.d/ | xargs sed -i 's/weekly/hourly/g' ; logrotate -f /etc/logrotate.conf"`

**Reply by anmr2380** [reply]
Is this command permanent? or restarting the cell phone it must be executed again... I am going to execute it

**Reply by DesktopECHO** [reply]
The change is permenant.

**Reply by anmr2380** [reply]
Tienes idea porque se genera este error en kern.log ???

May 15 21:11:31 a20 vmunix: [ 7091.230375]  [6:itor.ssh.client:15083] TCP: TCP: default window size: 523960

En messages 

May 15 21:13:06 a20 vmunix: [ 7185.541025]  [7:itor.ssh.client:15083] TCP: TCP: default window size: 523960

En syslog

May 15 21:12:29 a20 vmunix: [ 7148.763698]  [7:itor.ssh.client:15083] TCP: TCP: default window size: 523960

**Reply by DesktopECHO** [reply]
> 7:itor.ssh.client:15083

Do you have an SSH client installed on your phone?  If so, force-close it and see if the errors persist.  No idea otherwise.

**Reply by anmr2380** [reply]
Yes, I have ssh installed on my cell phone but I did everything until I uninstalled it and it seems that it is internal to pi deploy or debian, the problem persists

**Reply by DesktopECHO** [reply]
As far as I can determine with the info given, It's not an issue with Pi Deploy.  

You need to note the PID from the error message, then lookup the process name associated with the PID, then you'll know for sure. Search online for how to do this if you're not sure.
